frontends/mit-open/webpack.exports.js

```diff
@@ -0,0 +1,29 @@
+const path = require("path")
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
I added `webpack-merge` to `devDependencies` because that's where it belongs, and for some reason eslint doesn't respect that. I imagine that eslint ignores `webpack.config.js` but not other files that extend it.